app/helpers/sac_cas/event/participation_banner.rb

```diff
@@ -30,4 +30,21 @@ def render_cancel_popover
     @context.render('event/participations/popover_participant_cancels',
                     entry: @user_participation).to_s
   end
+
+  def status_text
+    t(@user_participation.state, scope: 'event.participations.states')
+  end
+
+  def status_class    
+    state = @user_participation.state.to_sym
+    alert_class = if %i[assigned attended summoned].include? state
+                    :success
+                  elsif %i[unconfirmed applied absent].include? state
+                    :warning
+                  else
+                    :danger
+                  end
```

Comment:
the other way of solving this would be to create a hash which maps the state to the alert class